### PR TITLE
[BALT-2023] Initial creation of Baltimore event

### PR DESCRIPTION
**Pull request by jaumann**
This is the initial placeholder for our 2023 event, which has the venue locked.

### CODE REVIEW

File: content/events/2023-baltimore/registration.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
++++
+Title = "Registration"
+Type = "event"
+Description = "Registration for devopsdays Baltimore 2020"
```
Comment (adrianmoisey) [inline content/events/2023-baltimore/registration.md:4]:
```suggestion
Description = "Registration for devopsdays Baltimore 2023"
```

**Comment by adrianmoisey** [comment]
Waiting on confirmation of any updated needed to mailing list and slack

File: data/events/2023-baltimore.yml [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+    image: "luke-samuels.jpg"
+  - name: "Jon Aumann"
+    employer: "Placemakr"
+    github: "jaumann"
+    linkedin: "https://www.linkedin.com/in/jaumann/"
+    twitter: "jaumann"
+    website: "https://placemakr.com"
+    image: "jon-aumann.jpg"
+  - name: "Charles Bushong"
+    employer: "Fearless Solutions"
```
Comment (bushong1) [inline data/events/2023-baltimore.yml:85]:
```suggestion
    employer: "Fearless"
```

File: data/events/2023-baltimore.yml [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+    website: "https://smartlogic.io"
+    image: "meghan-butler.jpg"
+  - name: "Luke Samuels"
+    role:  "Chairperson"
+    employer: "Fearless"
+    github: "degenerateddevops"
+    linkedin: "https://www.linkedin.com/in/lukesamuels/"
+    twitter: "degeneratedevop"
+    website: "http://fearless.tech"
+    image: "luke-samuels.jpg"
```
Comment (bushong1) [inline data/events/2023-baltimore.yml:76]:
```suggestion
```

**Review by bushong1 — COMMENTED** [review]
suggestions

**Comment by yvovandoorn** [comment]
@jaumann can you please look at the suggested changes? Otherwise I'll close this PR until one is submitted with the various changes suggested.

**Comment by yvovandoorn** [comment]
Hi @jaumann. I'm going to close this PR for now, but feel free to open up a new one when the team is ready to list Baltimore on devopsdays.org!

**Pull request closed without merging**